libs/core/tests/unit_tests/prompts/test_chat.py

```diff
@@ -709,3 +710,15 @@ async def test_messages_prompt_accepts_list() -> None:
 
     with pytest.raises(TypeError):
         await prompt.ainvoke([("user", "Hi there")])  # type: ignore
+
+
+def test_chat_input_schema() -> None:
```

Comment:
Could you add a unit test that shows the input variables for a chat prompt template for both scenarios?

Reply:
okay